lib/bap_disasm/bap_disasm_reconstructor.ml

```diff
@@ -48,6 +49,9 @@ let reconstruct name roots cfg =
   let find_block addr =
     Cfg.nodes cfg |> Seq.find ~f:(fun blk ->
         Addr.equal addr (Block.addr blk)) in
+  let find_callers blk =
```

Comment:
also, input edges of a block are not "callers", they are called predecessors. Moreover, there is the `[Cfg.Node.preds]` function for that. 

Please, learn the full interface of a module that you're using.